tasklist/src/services/notificationService.ts

```diff
@@ -0,0 +1,71 @@
+import {Http, Headers, RequestOptions, URLSearchParams} from '@angular/http';
+import 'rxjs/add/operator/map';
+import {Injectable} from "@angular/core";
+import { Observable } from 'rxjs/Rx';
+import { Storage } from '@ionic/storage';
+import { Config } from '../config/config';
+
+@Injectable()
+export class NotificationService {
+  public serverURL:string;
+  public propertiesURL:string;
+
+  constructor(public storage: Storage, protected http:Http, private _config:Config) {
+    this.serverURL = _config.get('apiUrl');
+    storage.get('user_id').then(
+      (user_id) => {
```

Comment:
I think it's an artefact of your first urls architecture on the API because you don't need the user_id in this line.